src/server/save.rs

```diff
@@ -388,11 +388,57 @@ impl SavedFile {
     }
 }
 
+/// Wrapper for the fields of an `Entries`. Stores text fields in a `Vec<(String, String)>` to
+/// enable accessing fields with duplicated names.
+
+#[derive(Debug, Default)]
+pub struct FieldsWrapper(Vec<(String, String)>);
```

Comment:
WRT the name, I think `TextFields` or something would be a little clearer.

Reply:
I agree `FieldsWrapper` is not the best name. I don't know about `TextFields` though; users might then expect the files stuff to be `FileFields`/`file_fields`. Maybe `FieldsVec` or just `Fields`?